### PR TITLE
Add documentation for message throw events

### LINKED ISSUE #146: Message Events Process is confusing for users
Copied over from the Zeebe repo: https://github.com/zeebe-io/zeebe/issues/4422

**Description**

The process shown at [Message Events](https://docs.zeebe.io/bpmn-workflows/message-events/message-events.html) is a bit confusing, since it seems the service task is directly completed and we wait until we receive a message which tells us that the money was collected.

We should try to re-model this process.

Related Zeebe Forum post 

https://forum.zeebe.io/t/is-the-zeebe-task-blocking-or-synchronous/1183

### PR DESCRIPTION
**Pull request by saig0**
* extend the BPMN reference section of message events
  * describe the usage of message throw events 
    * https://github.com/camunda-cloud/zeebe/issues/7763
    * https://github.com/camunda-cloud/zeebe/issues/7796
  * update the BPMN process and include a message end event
* update the BPMN coverage page and remove the duplicated manual task element

closes #146

### CODE REVIEW

File: docs/reference/bpmn-processes/message-events/message-events.md [hunk 1]
```diff
@@ -49,6 +47,31 @@ The `correlationKey` is an expression that usually [accesses a variable](/compon
 
 To correlate a message to the message event, the message is published with the defined name (e.g. `Money collected`) and the **value** of the `correlationKey` expression. For example, if the process instance has a variable `orderId` with value `"order-123"`, the message must be published with the correlation key `"order-123"`.
 
+## Message throw events
```
Comment (remcowesterhoud) [inline docs/reference/bpmn-processes/message-events/message-events.md:50]: 🔧  We should move this paragraph one paragraph up (above Messages), so it is grouped with the other events

File: docs/reference/bpmn-processes/message-events/message-events.md [hunk 2]
```diff
@@ -49,6 +47,31 @@ The `correlationKey` is an expression that usually [accesses a variable](/compon
...
+A process can contain intermediate message throw events or message end events to model the
+publication of a message to an external system, for example, to a Kafka topic.
+
+At the moment, intermediate message throw events and message end events behave exactly
+like [service tasks](../service-tasks/service-tasks.md) or [send tasks](../send-tasks/send-tasks.md)
+, and have the same job-related properties (e.g. job type, custom headers, etc.). The message throw
+events and the tasks are based on jobs
+and [job workers](../../../components/concepts/job-workers.md). The differences between the message
+throw events and the tasks are the visual representation and the semantics for the model. Read more
+about the job properties [here](../send-tasks/send-tasks.md#defining-a-task).
```
Comment (remcowesterhoud) [inline docs/reference/bpmn-processes/message-events/message-events.md:61]: ❓ We reference the send-tasks page here for the job properties. On that page is a reference to job type, task headers and variable mappings. However, it is not mentioned that these are job properties. Maybe we could reference them to the Job workers page which contains a clear list of job properties? The job worker page also mentions the key property.

File: media-src/product-manuals/zeebe/bpmn-processes/events/message-events.bpmn [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1jguk1q" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="0.7.0">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1jguk1q" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.9.0">
```
Comment (remcowesterhoud) [inline media-src/product-manuals/zeebe/bpmn-processes/events/message-events.bpmn:2]: ❓ Is this file used anywhere? I can't seem to find any usages for it.

**Review by remcowesterhoud — CHANGES_REQUESTED** [review]
👍 The text is good and understandable! Just a few minor suggestions:

🔧 We should add the job worker page to the references of the message-events page, as we now have a scenario where jobs need to be implemented for message events.
🔧 We should add an XML representation of the message throw event and message end event.

File: media-src/product-manuals/zeebe/bpmn-processes/events/message-events.bpmn (hunk 3 above)
Comment (saig0) [reply media-src/product-manuals/zeebe/bpmn-processes/events/message-events.bpmn:2]: The BPMN is not directly used. It is the source to generate the image that is used on the page. We try to keep these files in sync.

**Comment by saig0** [comment]
@remcowesterhoud I applied your hints. please have a second look :cookie: 

> :wrench: We should add an XML representation of the message throw event and message end event.

Yes, we could. But in most cases, the XML section is not relevant because most users use the modeler and don't look at the XML. Instead of making the XML section bigger, I would leave it as it is for now.

**Review by remcowesterhoud — DISMISSED** [review]
Looks good, feel free to merge 🚀

**Comment by saig0** [comment]
@remcowesterhoud please approve the PR again :sweat_smile: I rebased it.

**Review by remcowesterhoud — APPROVED** [review]
Re-approved 😄

**Pull request merged**